hm_1.py

```diff
@@ -0,0 +1,24 @@
+"""
+Calculate sum of all digits of a number set in string.
+
+Example:
+
+num = �12345�
+
+Your script should calculate it like: 1+2+3+4+5 = 15
+
+num = �611091234512�
+result = 6+1+1+0+9+1+2+3+4+6+1+2 = 36
+"""
+def sum_str(n):
+    sum_line = 0
+    for i in n:
```

Comment:
You can use the one-line command for this sum calculation.

Reply:
Replaced the best option